### DISCUSSION TITLE
How to get the client IP address

### DISCUSSION THREAD

**Discussion by Synseria** [abiosoft/colima#1166] [Q&A]
Hello,

I have a Mac Mini M1 with Colima.
I want to use this machine to expose my services on the internet (Vaultwarden, Plex, etc.). I've set up a reverse proxy (Traefik) in HOST mode (ports 80/443) to bridge between my domain name and my applications. It's working very well.

However, for security reasons, I want to read the IP addresses contacting my server, but I can't. I consistently receive 127.0.0.1.

I tried to get help from ChatGPT, but I couldn't solve my problem, leading me to wonder if it's even possible.

So, is it possible? If yes, how?

**Comment by abiosoft** [comment]
You mean IP address of the devices outside your network that is reaching the containers running in Colima?

**Reply by Synseria** [reply]
Yes, I want to see the IPs that come from the internet (But it can also come from my local network in 192.168.x.x)

**Reply by gabrielion** [reply]
did you solve it ? if yes how ?

**Reply by Synseria** [reply]
Yes, I installed Asahi Linux and started using Podman 🤡

Seriously, I spent a lot of time looking for a solution but couldn't find anything. By the way, if this issue is ever resolved, I would definitely consider switching back to Colima.
